.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+      // Use IntelliSense to learn about possible attributes.
+      // Hover to view descriptions of existing attributes.
+      // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Do JSON files support comments like this one?

Reply:
Yes, despite that weird highlighting on GitHub

Reply:
Yeah, VSCode uses JSON 2.0, which supports comments but had the same file extension as JSON 1.0.

Reply:
Ah, OK, thanks for explanation!